cmd/swapd/main.go

```diff
@@ -141,8 +142,9 @@ func cliApp() *cli.App {
 				Hidden: true, // flag is for integration tests and won't be supported long term
 			},
 			&cli.StringFlag{
-				Name:  flagEthereumEndpoint,
-				Usage: "Ethereum client endpoint",
+				Name:    flagEthereumEndpoint,
+				Usage:   "Ethereum client endpoint",
+				EnvVars: []string{"SWAPD_ETHEREUM_ENDPOINT"},
```

Comment:
I'd be in favor of abbreviating to ETH instead of ETHEREUM, but we'd want to shorten the flag name to match.

Reply:
that's fine to me

Reply:
I shortened it, but put in an alias that we can eventually eliminate for the old flag name, so people don't have to update their scripts immediately.